e2e/tests/caseManagementOrder_test.js

```diff
@@ -99,6 +156,32 @@ Scenario('local authority creates CMO', async (I, caseViewPage, draftCaseManagem
   I.seeAnswerInTab(10, 'Schedule', 'Key issues', 'Are there any other family or friends capable of caring in the children');
   I.seeAnswerInTab(11, 'Schedule', 'Parties\' positions', 'The mother agrees section 20');
   I.seeAnswerInTab(1, 'Recitals 1', 'Recital title', 'Recital 1');
-  await caseViewPage.goToNewActions(config.applicationActions.draftCaseManagementOrder);
-  await draftCaseManagementOrderEventPage.validatePreviousSelectedHearingDate('1 Jan 2050');
-});
+  I.seeAnswerInTab(6, 'Case management order', 'Is this ready to be sent to the judge?', cmoStatus);
+};
+
+const assertUserCannotSeeDraftOrders = async (I, userDetails) => {
+  await switchUserAndNavigateToCase(I, userDetails);
+  // Assert that it can't be seen
```

Comment:
These comments are bit unnecessary IMO. Code is pretty much readable, you made sure it is good. Not necessarily asking to remove it now but pointing that it can be avoided in the future.